src/runtime_src/core/include/xclbin.h

```diff
@@ -329,6 +329,20 @@ extern "C" {
         FAST_ADAPTER = 5
     };
 
+    // m_subtype
+    enum PS_SUBTYPE {
+        ST_PS = 0,
+        ST_DPU = 1,
+        ST_UNKNOWN = -1,
+    };
+
+    // m_functional
+    enum PS_FUNCTIONAL {
+        FC_DPU = 0,
+        FC_PREPOST = 1,
+        FC_UNKNOWN = -1,
```

Comment:
There should not be an unknown enumeration type. Please remove.

Reply:
removed